test/kt/CMakeLists.txt

```diff
@@ -122,3 +123,50 @@ if (ONEDPL_TEST_ENABLE_KT_ESIMD)
     # Pin some cases to track them, e.g. because they fail
     _generate_esimd_sort_test("256" "32" "double" "" 1000) # segfault
 endif()
+
+function (_generate_igpu_scan_test _data_per_work_item _work_group_size _type)
```

Comment:
Optional: It seems like we should be able to refactor this function and pull out some redundancy with `_generate_esimd_sort_test`.